pandas/core/series.py

```diff
@@ -1002,6 +1003,29 @@ def repeat(self, repeats, *args, **kwargs):
         return self._constructor(new_values,
                                  index=new_index).__finalize__(self)
 
+    def tile(self, reps, *args, **kwargs):
+        """
+        Tile elements of a Series. Refer to `numpy.tile`
+        for more information about the `reps` argument, although
+        note that we do not support multidimensional tiling of Series.
+
+        See also
+        --------
+        pd.Series.repeat
+        numpy.tile
+        """
+        nv.validate_tile(args, kwargs)
+        new_index = self.index.tile(reps)
+        if is_categorical_dtype(self.dtype):
+            new_values = Categorical.from_codes(np.tile(self.cat.codes, reps),
```

Comment:
do this more like ``repeats``, you just directly access the underlying function; then you don't need to type check